### ISSUE TITLE
Bug right shift

### ISSUE THREAD

**Issue by vranckxrobin** [verifast/verifast#323]
![shift fout](https://user-images.githubusercontent.com/47883033/197167447-f0cfb05c-a907-4f6b-853f-a513d21dde34.JPG)

I think this should not give an error.
Because for a shift it is only undefined behavior when the bits shifted is equal or greater then the number of bits that the variable represents.  In this case 1024LL is 64 bits so it should work with 63 bits.

**Comment by btj** [comment]
You are right; C11 6.2.6.2 paragraph 6 defines the width of an integer type as including the sign bit. This is also how CH2O and Cerberus interpret the standard. I will fix VeriFast.
